src/features/router/index.tsx

```diff
@@ -6,22 +6,22 @@ import ContactUsPage from 'src/features/contactUsPage';
 import AboutProjectPage from 'src/features/aboutProjectPage';
 
 import { MenuButton } from 'src/components/menuButton/menuButton';
-import { SwitchTheme } from 'src/components/switchTheme/switchTheme';
-
-import './index.css';
 
 import { ABOUT_US_PATH, CONTACT_US_PATH, FALLBACK_404_ROUTE, HISTORY_OBSERVER_PATH } from './path';
 
+import './index.css';
+
 const Router = () => (
   <BrowserRouter>
-    <MenuButton />
-    <SwitchTheme />
-    <Routes>
-      <Route path={HISTORY_OBSERVER_PATH + ':data?'} element={<HistoryPage />} />
-      <Route path={ABOUT_US_PATH} element={<AboutProjectPage />} />
-      <Route path={FALLBACK_404_ROUTE} element={<Page404 />} />
-      <Route path={CONTACT_US_PATH} element={<ContactUsPage />} />
-    </Routes>
+    <div className="container">
```

Comment:
are you sure we need this one here?

Reply:
Yes, as we discussed it before

Reply:
let's delete it for now. You will see later why